test/e2e/test_02_server.py

```diff
@@ -98,6 +98,11 @@ def test_deny(self, manager):
         manager.expect_status_code(
             "/knora/DenyLeaves.jpg/full/max/0/default.jpg", 401)
 
+    def test_not_found(self, manager):
+        """return 401 Unauthorized if the user does not have permission to see the image"""
+        manager.expect_status_code(
+            "/file-should-be-missing-123", 404)
```

Comment:
*question* what should the test actually check? 

I assume the description should talk about a 404 and not an unauthorized...

Reply:
Mindless focus, sorry. Will fix in a second.

Reply:
Fixed.